tests/baselines/reference/subclassThisTypeAssignable.errors.txt

```diff
@@ -25,10 +41,23 @@ tests/cases/compiler/tile1.ts(6,81): error TS2744: Type parameter defaults can o
     interface MyAttrs { id: number }
     class C implements ClassComponent<MyAttrs> {
     	view(v: Vnode<MyAttrs>) { return 0; }
+    	~~~~
+!!! error TS2416: Property 'view' in type 'C' is not assignable to the same property in base type 'ClassComponent<MyAttrs>'.

```

Comment:
This basically comes down to the following break:

```ts
interface Indexable {
    [x: number]: any;
}

class C implements Indexable {
}
```

In earlier versions, implementing `C` without re-declaring the index signature was valid. I would argue that this behavior is still surprising or incorrect for string index signatures.